### PR TITLE
[BLOG] Add the-ibis-backends

### PR DESCRIPTION
**Pull request by MarsBarLee**
## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. ([link to details in updated Labs blog migration process document](https://www.notion.so/quansightlabs/Labs-blog-migration-process-200c3297b67245b3953dd23487badfe9#af61137685c849c0affcdbab853282a7))

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by MarsBarLee** [comment]
@gabalafou This seems to have a similar issue with the MDX parser, tables and inline styling:
```
error [Error: [next-mdx-remote] error compiling MDX:
Unexpected closing slash `/` in tag, expected an open tag first
  55 | for different backends over a span of six years.
  56 |
> 57 | |  <div style="width:120px"/></div>  | <div style="width:70px"/>&nbsp; &nbsp; 2015</div> | <div style="width:70px"/>&nbsp; &nbsp; 2016</div> | <div style="width:70px"/>&nbsp; &nbsp; 2017</div> | <div style="width:70px"/>&nbsp; &nbsp; 2018</div> | <div style="width:70px"/>&nbsp; &nbsp; 2019</div> | <div style="width:70px"/>&nbsp; &nbsp; 2020</div> |
     |      
  58 | |:-------------|:------:|:------:|:------:|:------:|:------:|:------:|
  59 | | [Omnisci]    |        |        |        |   31   |   33   |   38   |
```
[Link to this Vercel build error](https://vercel.com/quansight/labs/2v5M2JpjA8rKH4vKdhC6d3wd9BbV#:~:text=12%3A14%3A55.823-,error%20%5BError%3A%20%5Bnext%2Dmdx%2Dremote%5D%20error%20compiling%20MDX%3A,-12%3A14%3A55.823):

Should I remove the styling, like you did with [jupyter-community-calls](https://github.com/Quansight/Quansight-website/pull/388#issuecomment-1256809876)? Or would it take more time, like [array-libraries-interoperability](https://github.com/Quansight/Quansight-website/pull/391#issuecomment-1256804183)?

File: apps/labs/posts/the-ibis-backends.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+We'll unravel some of the different capabilities of each approach below.
+
+## A data-driven history of Ibis compatibility
+
+The table below looks at over 2000 issues in the Ibis project.
+It provides an annual summary of the issues tagged in Ibis
+for different backends over a span of six years.
+
+|  <div style="width:120px"/></div>  | <div style="width:70px"/>&nbsp; &nbsp; 2015</div> | <div style="width:70px"/>&nbsp; &nbsp; 2016</div> | <div style="width:70px"/>&nbsp; &nbsp; 2017</div> | <div style="width:70px"/>&nbsp; &nbsp; 2018</div> | <div style="width:70px"/>&nbsp; &nbsp; 2019</div> | <div style="width:70px"/>&nbsp; &nbsp; 2020</div> |
```
Comment (gabalafou) [inline apps/labs/posts/the-ibis-backends.md:57]:
```suggestion
|  <div style="width:120px"></div>  | <div style="width:70px">&nbsp; &nbsp; 2015</div> | <div style="width:70px">&nbsp; &nbsp; 2016</div> | <div style="width:70px">&nbsp; &nbsp; 2017</div> | <div style="width:70px">&nbsp; &nbsp; 2018</div> | <div style="width:70px">&nbsp; &nbsp; 2019</div> | <div style="width:70px">&nbsp; &nbsp; 2020</div> |
```

File: apps/labs/posts/the-ibis-backends.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
+---
+title: 'Querying multiple backends with Ibis'
+published: 2020-11-13
```
Comment (gabalafou) [inline apps/labs/posts/the-ibis-backends.md:3]:
```suggestion
published: November 13, 2020
```

**Comment by gabalafou** [comment]
@MarsBarLee, I think your instinct to remove styling is correct. I noticed some other errors, too:

- Unclosed tag
  - Wrong: `<br>`
  - Correct: `<br />`
- Bad date format
  - Wrong: `2020-11-13`
  - Correct: `November 13, 2020`
- Single quotes in single quotes
  - Wrong: `'We'll look into ways to write better.'`
  - Correct: `"We'll look into ways to write better."`

I pushed some commits to remove the inline styles and fix the other errors. 

I think it looks okay now, you agree?

**Comment by MarsBarLee** [comment]
@gabalafou Thank you for reviewing and making those fixes! Yes, it looks better now.

I'll keep note about the unclosed tags. I knew about `` so I watch out for `<b/>` as well. The dates format and quotes as well.

**Comment by gabalafou** [comment]
@MarsBarLee want to take this PR out of draft mode?

**Comment by MarsBarLee** [comment]
@gabalafou Yes, thank you for reminding me! It's marked as ready to review.

**Review by gabalafou — APPROVED** [review]

**Comment by gabalafou** [comment]
Approved 😃 

@MarsBarLee would you please squash and merge?

**Comment by gabalafou** [comment]
If you can't merge the PR into the `develop` branch, then I'll do it 😌

**Comment by MarsBarLee** [comment]
@gabalafou Thanks for the review! Successfully squashed and merged into the `develop` branch!

**Pull request merged**